py_mmd_tools/mmd_to_adc.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+'''
+This script generates a yaml file contatining requiremnents specific to the Arctic Data Centre.
+This new yaml file is generated using the more general mmd_elements.yaml file.
```

Comment:
```suggestion
This new yaml file is generated using the more general adc_elements.yaml file.
```
?

Reply:
mmd_elements.yaml is the input file, whereas adc_elements.py is the new output.

Reply:
Here also, I would go for strict separation between the standards. In this case, we would like to translate from MMD to ACDD, wouldn't we? You could make similar tools like `mmd_to_cf.py` etc.